pandas/io/formats/format.py

```diff
@@ -179,15 +180,19 @@ def __init__(self, series, buf=None, length=True, header=True, index=True,
 
     def _chk_truncate(self):
         from pandas.core.reshape.concat import concat
+        min_rows = self.min_rows
```

Comment:
can you add some comments on what the procedure is here

Reply:
done